v2.0/configure-replication-zones.md

```diff
@@ -15,18 +15,36 @@ This page explains how replication zones work and how to use the `cockroach zone
 
 ## Overview
 
-### Replication Zone Levels
+### Pre-Configured Replication Zones
 
-There are four replication zone levels:
+CockroachDB comes with pre-configured replication zones for all user data in the cluster and for some internal system data. These replication zones can be adjusted but not deleted.
 
-- **Cluster:** CockroachDB comes with a single, default replication zone for the entire cluster. See [View the Default Replication Zone](#view-the-default-replication-zone) and [Edit the Default Replication Zone](#edit-the-default-replication-zone) for more details.
-- **Database:** You can add replication zones for specific databases. See [Create a Replication Zone for a Database](#create-a-replication-zone-for-a-database) for more details.
-- **Table:** You can add replication zones for specific tables. See [Create a Replication Zone for a Table](#create-a-replication-zone-for-a-table) for more details.
-- **Row:** ([For enterprise users](enterprise-licensing.html)) You can add replication zones for specific rows in a table by [defining table partitions](partitioning.html). See [Create a Replication Zone for a Table Partition](#create-a-replication-zone-for-a-table-partition-new-in-v2-0) for more details.
+Zone Name | Description
+----------|------------
+`.default` | This replication zone applies to all user data in the cluster not constrained by more specific, [user-created replication zones](#user-created-replication-zones).
+`.meta` | The "meta" ranges contain the authoritative information about the location of all data in the cluster. If these ranges are unavailable, the entire cluster will be unavailable, so this replication zone is pre-configured to have a lower-than-default `ttlseconds`. If your cluster is running in multiple datacenters, it's a best practice to configure the meta ranges to have a copy in each datacenter.
+`.liveness` | <span class="version-tag">New in v2.0:</span> The "liveness" range contains the authoritative information about which nodes are live at any given time. If this range is unavailable, the entire cluster will be unavailable, so this replication zone is pre-configured to have a lower-than-default `ttlseconds`. Giving it a high replication factor is also strongly recommended.
```

Comment:
> If this range is unavailable, the entire cluster will be unavailable, so this replication zone is pre-configured to have a lower-than-default `ttlseconds`.

That doesn't logically follow. The lower-than-default `ttlseconds` is because historical queries are never run on the liveness table and its advantageous to keep the liveness range smaller for reliable performance.

The "If this range is unavailable, the entire cluster will be unavailable" bit is a reason to recommend running it with a higher-than-default replication factor.

Reply:
Thanks for the clarification. Reworked.